submissions/anthoniaAdollo/html-forms/contactForm.html

```diff
@@ -63,18 +61,14 @@
         <li><a href="https://www.instagram.com">Instagram</a></li>
         <li><a href="https://www.facebook.com">Facebook</a></li>
     </ul>
-
 <footer>
     <nav>
-        <a href="#">Home</a>
-        <a href="#"> New Stock</a>
-        <a href="#">Sign Up</a>
-        <a href="#">Sale</a>
+        <a href="/html-forms/contactForm.html">Contact Us</a>
```

Comment:
You can simply do this `href="contactForm.html"` It will work fine because it is on the same folder